when ever i run script/bootstrap i receive this

when ever i run script/bootstrap or npm install i receive this.
```
npm ERR! git clone git@github.com:github:maddox/harmonyhubjs-client Cloning into bare repository '/root/.npm/_git-remotes/git-github-com-github-maddox-harmonyhubjs-client-def8ff3d'...
npm ERR! git clone git@github.com:github:maddox/harmonyhubjs-client Permission denied (publickey).
npm ERR! git clone git@github.com:github:maddox/harmonyhubjs-client fatal: Could not read from remote repository.
npm ERR! git clone git@github.com:github:maddox/harmonyhubjs-client
npm ERR! git clone git@github.com:github:maddox/harmonyhubjs-client Please make sure you have the correct access rights
npm ERR! git clone git@github.com:github:maddox/harmonyhubjs-client and the repository exists.
npm ERR! notarget No compatible version found: harmonyhubjs-client@'github:maddox/harmonyhubjs-client#for-harmony-api'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["1.1.2","1.1.3","1.1.4","1.1.5","1.1.6","1.1.7","1.1.8","1.1.9","1.1.10"]
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! System Linux 4.9.80-v7+
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! cwd /home/pi/Downloads/Harmony/harmony-api
npm ERR! node -v v4.8.2
npm ERR! npm -v 1.4.21
npm ERR! code ETARGET
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /home/pi/Downloads/Harmony/harmony-api/npm-debug.log
npm ERR! not ok code 0

```

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "express": "^4.12.4",
     "morgan": "^1.6.1",
     "body-parser": "^1.12.4",
-    "harmonyhubjs-client": "maddox/harmonyhubjs-client#for-harmony-api",
+    "harmonyhubjs-client": "^1.1.0",
     "harmonyhubjs-discover": "^1.0.2",
     "parameterize": "^0.0.7",
     "mqtt": "^1.4.1"

